src/main/interface/tramp_protocol.c

```diff
@@ -0,0 +1,97 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <string.h>
+#include "platform.h"
+#include "common/utils.h"
+#include "interface/tramp_protocol.h"
+
+#define TRAMP_SYNC_START            0x0F
+#define TRAMP_SYNC_STOP             0x00
+#define TRAMP_COMMAND_SET_FREQ      'F' // 0x46
+#define TRAMP_COMMAND_SET_POWER     'P' // 0x50
+#define TRAMP_COMMAND_ACTIVE_STATE  'I' // 0x49
+#define TRAMP_COMMAND_GET_CONFIG    'v' // 0x76
+
+uint8_t trampCrc(const trampFrame_t *frame);
+void trampFrameInit(uint8_t frameType, trampFrame_t *frame);
+void trampFrameClose(trampFrame_t *frame);
+
```

Comment:
No need for fwd declarations. If those functions are public, they should be declared in the header file. If they are private please remove the decalration and add `static` to the actual function implementaion. In a protocol you should be very clear on what is public and what is private.

Reply:
I'll clean these up.